huntsman/tests/test_huntsman.py

```diff
@@ -180,6 +180,17 @@ def test_default_lookup_trigger(pocs):
     assert pocs._lookup_trigger() == 'parking'
 
 
+def test_darks_collection(pocs):
+    pocs.state = 'observing'
+
+    pocs.is_dark(horizon="flat") is True
```

Comment:
```suggestion
    pocs.is_dark(horizon="flat") = True
```

Reply:
Yes, I am setting up the conditions for the dark state. I tried using `=` but got an error which says "can't assign to function call". That's why I used instead `is`. Should I ignore that error?